electron/wire.json

```diff
@@ -0,0 +1,19 @@
+{
+  "adminUrl": "https://teams.wire.com",
+  "appBase": "https://app.wire.com",
+  "buildNumber": "0",
+  "copyright": "© Wire Swiss GmbH",
+  "customProtocolName": "wire",
+  "description": "The most secure collaboration platform.",
+  "environment": "internal",
+  "legalUrl": "https://wire.com/legal/",
+  "licensesUrl": "https://wire.com/legal/licenses/",
+  "maximumAccounts": "3",
```

Comment:
Let's put this as a number instead of a string

Reply:
Since it will be injected as a string when loading a configuration via `process.env`, I figured we should save it as a string here, too.